game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/CombatValue.java

```diff
@@ -0,0 +1,98 @@
+package games.strategy.triplea.delegate.power.calculator;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.TerritoryEffect;
+import games.strategy.engine.data.Unit;
+import games.strategy.triplea.delegate.Matches;
+import java.util.Collection;
+
+public interface CombatValue {
+
+  StrengthAndRollCalculator getRoll();
+
+  StrengthAndRollCalculator getStrength();
+
+  boolean isDefending();
+
+  GameData getGameData();
+
+  static CombatValue buildMain(
```

Comment:
`buildMain` is a bit unclear here. It seems implied this is `buildMainCombat`, in which case I think that should be explicitly named to keep it easier on the reader.

Reply:
`CombatValue.buildMainCombatValue()` feels a little redundant so that's why I just left it as `CombatValue.buildMain` but I'll change the name.